packages/popcorn/src/utils/interpolate.ts

```diff
@@ -102,14 +117,14 @@ export default <T>(
   );
 
   invariant(
-    !ease || !Array.isArray(ease) || ease.length === input.length - 1,
+    !ease || !Array.isArray(ease) || ease.length === inputLength - 1,
     'Array of easing functions must be of length `input.length - 1`, as it applies to the transitions **between** the defined values.'
   );
 
   // If input runs highest -> lowest, reverse both arrays
   if (input[0] > input[inputLength - 1]) {
-    input = [...input];
-    output = [...output];
+    input = [].concat(input);
```

Comment:
Seems like TS rest operator is kinda broken, for `number[] | string[]` it makes `(number | string)[]` and that cannot be assigned back to the original output type 

https://www.typescriptlang.org/play/#src=function%20interpolate(output%3A%20number%5B%5D%20%7C%20string%5B%5D)%20%7B%0D%0A%20%20%20%20const%20spread%20%3D%20%5B...output%5D%0D%0A%20%20%20%20output%20%3D%20spread%20%20%0D%0A%7D%0D%0A